novawallet/Modules/Swaps/Setup/View/SwapSetupViewLayout.swift

```diff
@@ -27,6 +27,26 @@ final class SwapSetupViewLayout: ScrollableContainerLayoutView {
         $0.imageWithTitleView?.iconImage = R.image.iconActionSwap()
     }
 
+    let detailsHeaderCell: CollapsableViewHeader = .create {
```

Comment:
I think we can just implement the behaviour with collapsing using the same approach as in NetworkInfoView that is used in Staking. But I would use StackTableView there instead of UIStackView that is used in NetworkInfoView.